docs/docs/pods.md

```diff
@@ -133,7 +149,7 @@ Use the `/v2/pods/` endpoint to create and manage your pods. [See the full API s
 ## Create
 
 ```json
- $ curl POST <ip>:<port>/v2/pods <mypod>.json
+ $ curl -X POST -H "Content-type: application/json" -d@/dev/stdin http://<ip>:<port>/v2/pods <mypod>.json
```

Comment:
if there's a pod json file, then the syntax is `-d@{path-to-pod-json-file}`. in this case that's `-d@<mypod>.json`